apps/masterbots.ai/types/types.ts

```
@@ -16,6 +17,12 @@ export interface Chat extends Record<string, any> {
   sharePath?: string
 }
 
+export type AiToolCall = {
+  toolCallId: string
+  toolName: WordWareFlowPaths
+  args: Record<string, any>
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)